packages/README.md

```diff
@@ -0,0 +1,14 @@
+# Cal-ITP Python Packages
+
+We publish calitp-data-analysis and calitp-data-infra, two Python packages that enable work elsewhere in the repository. Each is maintained and deployed the same way.
+
+## Testing Changes
+
+Each package should pass mypy and other static checkers, and each has a small
+number of tests. These checks are executed in GitHub Actions when a PR is opened.
+
+## Deploying Changes to Production
+
+When changes are finalized, a new version number should be specified in the relevant package's [pyproject.toml](./pyproject.toml) file. When changes to the package directory are merged into `main`, the relevant GitHub Action (`build-calitp-data-analysis` or `build-calitp-data-infra`) automatically publishes an updated version of the package to PyPI. A contributor with proper PyPI permissions can also manually release a new version of the targeted package via the CLI, or test a release using [TestPyPI](https://packaging.python.org/en/latest/guides/using-testpypi/).
```

Comment:
Is it worth specifying how to give someone the right PyPI permissions?

Reply:
Addressed in [ac52c02](https://github.com/cal-itp/data-infra/pull/3156/commits/ac52c0287d433386871beca0c45af14ede6b1f84) - though there's no public-facing docs about how to do it, just menus once a project owner is logged into PyPI.